databases/usersessions/dataGenerator.js

```diff
@@ -0,0 +1,158 @@
+/* ===========================================================
+Dependencies
+=========================================================== */
+const pg = require('pg');
+const Promise = require('bluebird');
+
+const Chance = require('chance');
+const chance = new Chance();
+
+const topTraders = require('./topTraders.js');
+const config = require('./config.js');
+
+/* ===========================================================
+Connect to PostgreSQL with Pool, using settings in config.js
+=========================================================== */
+
+const pool = new pg.Pool(config);
+
+pool.on('error', (err) => {
```

Comment:
You can omit the braces + new line here (per [airbnb style guide](http://airbnb.io/javascript/#arrows--implicit-return)